builtin/mainmenu/textures.lua

```diff
@@ -145,6 +145,17 @@ function mm_texture.set_game(identifier,gamedetails)
 	
 	local path = gamedetails.path .. DIR_DELIM .."menu" ..
 									 DIR_DELIM .. identifier .. ".png"
+	if core.setting_getbool("random_menu_images") == true then
+		local count = gamedetails.image_count
+		if tonumber(count) == nil then
+			count = 1
+		end
+		local new_path = path .. "." .. math.random(1, count)
+		if io.open(new_path,"r") ~= nil and math.random(1, (count+1)) ~= 1 then
```

Comment:
That was the method I originally tried, but I couldn't figure out how to do it so that the first image (without a number after) wasn't shown more often that the others...